spec/emittr_spec.rb

```diff
@@ -101,6 +101,101 @@
     end
   end
 
+  describe '#on_any' do
+    let(:block)     { Proc.new {} }
+    let(:any_block) { Proc.new {} }
+
+    context 'when listeners for a specific event are set' do
+      it 'calls #on_any blocks' do
+        emitter.on_any &any_block
+
+        emitter.on :first_any_test, &block
+        emitter.on :second_any_test, &block
+
+        expect(any_block).to receive(:call).with(:first_any_test).twice
+        expect(any_block).to receive(:call).with(:second_any_test).twice
+
+        2.times { emitter.emit :first_any_test }
+        2.times { emitter.emit :second_any_test }
+      end
+    end
+
+    context 'when no events are set' do
+      it 'calls #on_any blocks' do
+        emitter.on_any &any_block
+
+        expect(any_block).to receive(:call).with(:on_any_test).twice
+
+        2.times { emitter.emit :on_any_test }
+      end
+    end
+  end
+
+  describe '#off_any' do
+    let(:block) { Proc.new {} }
+    let(:block_for_on) { Proc.new {} }
```

Comment:
This one too, we should create it inside the `it` below since it's not used inside the whole `describe`.